Tests/helper.py

```diff
@@ -348,6 +349,10 @@ def on_ci():
     )
 
 
+def is_big_endian():
+    return platform.processor() == "s390x"
```

Comment:
There is a constant:
https://docs.python.org/3/library/sys.html#sys.byteorder

Reply:
Thanks, updated!